contracts/erc734/ERC734.sol

```diff
@@ -1,21 +1,54 @@
 // SPDX-License-Identifier: GPL-3.0
 pragma solidity ^0.6.9;
 
-import "./storage/Storage.sol";
-import "./library/LibraryLock.sol";
-import "./interface/IERC734.sol";
-
+import "../interface/IERC734.sol";
 
 /**
  * @dev Implementation of the `IERC734` "KeyHolder" interface.
  */
-contract ERC734 is Storage, LibraryLock, IERC734 {
-    event ExecutionFailed(uint256 indexed executionId, address indexed to, uint256 indexed value, bytes data);
+contract ERC734 is IERC734 {
+    /**
+    * @dev Definition of the structure of a Key.
+    *
+    * Specification: Keys are cryptographic public keys, or contract addresses associated with this identity.
+    * The structure should be as follows:
+    *   - key: A public key owned by this identity
+    *      - purposes: uint256[] Array of the key purposes, like 1 = MANAGEMENT, 2 = EXECUTION
+    *      - keyType: The type of key used, which would be a uint256 for different key types. e.g. 1 = ECDSA, 2 = RSA, etc.
+    *      - key: bytes32 The public key. // Its the Keccak256 hash of the key
+    */
+    struct Key {
```

Comment:
both structs should be included in Storage in order to respect the variables and functions order